tests/scripts/github-action-helper.sh

```diff
@@ -218,24 +218,46 @@ DNS.4 = ${SERVICE}.${NAMESPACE}.svc.cluster.local
 IP.1  = ${IP}
 EOF
 
-  openssl req -new -key "${DIR}"/"${SERVICE}".key -subj "/CN=${SERVICE}.${NAMESPACE}.svc" -out "${DIR}"/server.csr -config "${DIR}"/csr.conf
```

Comment:
Are the change in this script the same as the first script, or are there any differences? If the same, could it be factored to a new script that is called by the two scripts?

Reply:
I'm referring to this whole section of script that seems the same in two different scripts, if it could be factored out.

Reply:
yes okay, it can be  factored out,
just some small addition is there for the `generate_vault_tls_config function`  so I will add it in another function.